widgetssdk/src/main/java/com/glia/widgets/survey/SurveyView.java

```diff
@@ -68,6 +78,48 @@ public SurveyView(@NonNull Context context, @Nullable AttributeSet attrs, int de
         initAdapter();
     }
 
+    @Override
+    protected void onLayout(boolean changed, int left, int top, int right, int bottom) {
+        super.onLayout(changed, left, top, right, bottom);
+        applyStyle(theme.getSurveyStyle());
+    }
+
+    private void applyStyle(SurveyStyle surveyStyle) {
+        cardView.setCardBackgroundColor(Color.parseColor(surveyStyle.getLayer().getBackgroundColor()));
+        ResourceProvider resourceProvider = Dependencies.getResourceProvider();
+        float cornerRadius = resourceProvider.convertDpToPixel(surveyStyle.getLayer().getCornerRadius());
+        cardView.setRadius(cornerRadius);
+        ViewGroup.MarginLayoutParams layoutParams =
+                (ViewGroup.MarginLayoutParams) cutBottomCornersCardView.getLayoutParams();
+        int marginTopDp = resourceProvider.getDimension(R.dimen.glia_x_large);
+        int marginTopPx = (int) resourceProvider.convertDpToPixel(marginTopDp);
+        layoutParams.setMargins(
```

Comment:
"shapeAppearanceOverlay" could be better solution for that (override CardView style)?

Reply:
```
            <item name="cornerFamily">rounded</item>
            <item name="cornerSizeTopRight">cornerRadius</item>
            <item name="cornerSizeTopLeft">cornerRadius</item>
```

Reply:
@deniss-mc Yes, that could be a better solution. The matter is that we need to provide an ability to change corner radius programmatically in runtime. And I'm not sure it's possible with the "shapeAppearanceOverlay" approach.

Do you think it's possible?

Reply:
Thank you, changed the implementation in 15e0942a8574618796e33ea646d3ecca55488008